RELEASE_NOTES.md

```diff
@@ -1,5 +1,10 @@
 # Anubis Release Notes
 
+### Change the main py in order to work with the frontend app
```

Comment:
follow the existing structure, don't mess up.
1 - cors is not in this pr, so you can remove it.
2 - simply add the change description to real-export (what was changed), under new features in this case for 0.0.3